docs/developer/release/prerequisites.md

```diff
@@ -0,0 +1,29 @@
+# Prerequisites
+
+Before performing a release, make sure that you have completed the following.
+
+## Steps
+
+1. Install [helm-docs](https://github.com/norwoodj/helm-docs) on mac/linux.
+
+    This will be used when updating helm charts.
+
+2. Follow the GitHub [instructions](https://docs.github.com/en/authentication/managing-commit-signature-verification) to set up GPG for signature verification.
+
+*NOTE: This is required for creating tags on the release branch.*
```

Comment:
It's not listed as optional, so I think we don't need to call it out as required.

```suggestion
```